src/App/ProjectDetail/ProjectDetailController.java

```diff
@@ -184,6 +184,18 @@ private void onreleaseID(KeyEvent keyEvent) {
     private void showDays(ActionEvent event) throws IOException {
         getEsti_time();
 
+        if (start_date.getValue() != null) {
+            final Date startDate = Date.valueOf(start_date.getValue());
+            Date endDate = Date.valueOf(end_date.getValue());
+            if (endDate.compareTo(startDate) < 0) {
+                Label label = new Label("End date must be equal or greater than start date");
+                JFXPopup popup = new JFXPopup(label);
+                popup.show(end_date, JFXPopup.PopupVPosition.BOTTOM, JFXPopup.PopupHPosition.RIGHT);
+                end_date.setValue(null);
```

Comment:
I think you should not set ```end_date``` to ```null``` since in the next line you have placed the ```return``` keyword which eventually checking the above if condition again and ``` Date endDate``` is being assigned to ```null```, so that gives a null pointer exception error.

Reply:
What should I show if the user hasn't selected a start date, yet he chooses an end date?
The same `End date must be equal or greater than start date` label?

Reply:
Yes, I think we can use this statement (```End date must be equal or greater than start date```). Also, if we want we can use another statement i.e. ```Start date cannot be empty```. I leave it to your decision :)